ci-operator/config/openshift/cluster-cloud-controller-manager-operator/openshift-cluster-cloud-controller-manager-operator-release-4.9.yaml

```diff
@@ -83,15 +83,6 @@ tests:
   steps:
     cluster_profile: azure4
     workflow: openshift-e2e-azure-ccm-install
-- as: e2e-openstack-ccm
```

Comment:
I think we should be keeping this one right?

Reply:
If you think there's a chance we'll backport changes to CCM in 4.9, then yeah. What's the support status of CCCMO in 4.9?

Reply:
The docs on 4.9 mention support as Tech preview for openstack, perhaps it makes sense?

https://docs.openshift.com/container-platform/4.9/operators/operator-reference.html#cluster-cloud-controller-manager-operator_platform-operators-ref

Reply:
I agree. Done